python/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+source .env
```

Comment:
@vpchung rather than passing environment variables into the container via `docker run -v ...`, you should be able to pass them as an `env-file`: https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file

So, the command would be:
```
docker run --rm \
    -env-list .env \
    -v $PWD/output:/output:rw \
    pubmed_crawler
```
... and your variables would automatically be set (no need to `source .env`).

You might need to tweak the `.envTemplate` format a bit, such that it looks more like this (e.g., no `export` commands):
```
###########################
# Entrez Credentials     #
###########################
# Providing a valid email and API key will allow up to 10 requests
# per second. You may experience errors if this information is not
# provided.
ENTREZ_EMAIL=email
ENTREZ_API_KEY=apikey
```

I think this would also let you make usage of the container a bit more flexible (i.e., passing in arguments via `docker run`), but it's fine to save that for a different PR.